### PR TITLE
NoOp implementation of Resiliency

### PR DESCRIPTION
**Pull request by pkedy**
# Description

Adding a NoOp implementation of Resiliency for true bypass behavior.

/cc: @artursouza @halspang @berndverst

### CODE REVIEW

File: pkg/resiliency/noop.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package resiliency
```
Comment (artursouza) [inline pkg/resiliency/noop.go:1]: license headers.

**Review by artursouza — CHANGES_REQUESTED** [review]

File: pkg/resiliency/noop.go (hunk 1 above)
Comment (pkedy) [reply pkg/resiliency/noop.go:1]: Thanks!

File: pkg/resiliency/noop_test.go [hunk 2]
```diff
@@ -0,0 +1,55 @@
+package resiliency
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNoOp(t *testing.T) {
```
Comment (halspang) [inline pkg/resiliency/noop_test.go:23]: Should the no-op policy have tests that show it is truly a no-op (return failure and not retry).

Comment (pkedy) [reply pkg/resiliency/noop_test.go:23]: Yeah I'll tweak the tests to return an error for each call type.

**Review by halspang — APPROVED** [review]

**Comment by yaron2** [comment]
Thanks for this PR, true bypass is extremely important. We learned it the hard way early on with tracing.

**Review by yaron2 — APPROVED** [review]

**Review by berndverst — APPROVED** [review]

**Review by artursouza — APPROVED** [review]

**Pull request merged**